index.html

```diff
@@ -2204,33 +2219,33 @@ <h3>Extensibility</h3>
 
         <ul>
           <li>
-Model complex multi-entity relationships through the use of a
-<a>graph</a>-based data model.
+Model complex multi-entity relationships through the use of a <a>graph</a>-based
+data model.
           </li>
           <li>
 Extend the machine-readable vocabularies used to describe information in the
 data model, without the use of a centralized system for doing so, through the
 use of [[?LINKED-DATA]].
           </li>
           <li>
-Support multiple types of cryptographic proof formats through the use of
-Linked Data Proofs [[?LD-PROOFS]], Linked Data Signatures [[?LD-SIGNATURES]],
-and a variety of signature suites.
+Support multiple types of cryptographic proof formats through the use of Linked
+Data Proofs [[?LD-PROOFS]], Linked Data Signatures [[?LD-SIGNATURES]], and a
+variety of signature suites.
           </li>
           <li>
-Provide all of the extensibility mechanisms outlined above in a data format
-that is popular with software developers and web page authors, and is enabled
-through the use of [[JSON-LD]].
+Provide all of the extensibility mechanisms outlined above in a data format that
+is popular with software developers and web page authors, and is enabled through
+the use of [[JSON-LD]].
           </li>
         </ul>
 
         <p>
-This approach to data modelling is often called an
+This approach to data modelling is often called an 
```

Comment:
```suggestion
This approach to data modeling is often called an 
```

(American spelling preferred in W3C specs)